src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -579,12 +604,47 @@ public FlagUsageException(String message) {
    *
    * @param files A list of filenames.
    * @param zips A list of zip filenames.
+   * @param jsonFiles A list of json encoded files.
+   * @return An array of inputs
+   */
+  protected List<SourceFile> createInputs(List<String> files,
+      List<String> zips, List<JsonFileSpec> jsonFiles) throws FlagUsageException, IOException {
```

Comment:
OK, then it's fine to combine json and files. But the zips are currently used only for some rare usecase internally, so for now I think it's safe to assume we don't need to combine zips and json inputs. So you can delete the zips argument.